app/static/css/style.css

```diff
@@ -1,22 +1,27 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;700;900&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200;300;400;600;700&display=swap');
```

Comment:
Any reason we're still keeping Nunito around?

Reply:
nope haha I'll get rid of it